imap/deliver.c

```diff
@@ -240,6 +240,7 @@ int main(int argc, char **argv)
     prot_settimeout(deliver_in, 300);
 
     cyrus_init(alt_config, "deliver", CYRUSINIT_NODB, CONFIG_NEED_PARTITION_DATA);
+    global_sasl_init(1, 0, NULL);
```

Comment:
I'm pretty sure it's actually necessary for some configurations or else I wouldn't have added it way back when I did.

Reply:
My gut feeling is that it was probably necessary long ago for some reason like "otherwise the array is left uninitialised and full of rubbish", but perhaps isn't necessary for that purpose anymore.

But anyway this part is fine, I was just commenting, not requesting a change.